gherkin/php/src/PickleCompiler.php

```diff
@@ -184,10 +195,17 @@ private function pickleStep(Step $step, array $variableCells = [], ?TableRow $va
             $argument = null;
         }
 
+        $this->lastKeywordType =
+            $step->keywordType === Step\KeywordType::CONJUNCTION
+            ? $this->lastKeywordType
+            : $step->keywordType ?? Step\KeywordType::UNKNOWN
```

Comment:
Maybe parentheses here for clarity? I can never remember the order of operations